wagtail/admin/templates/wagtailadmin/shared/keyboard_shortcuts_dialog.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailadmin_tags i18n %}
+{% dialog icon_name='regex' id='keyboard-shortcuts-dialog' title=_("Keyboard shortcuts") %}
+    <table class="w-keyboard-shortcuts">
```

Comment:
```suggestion
    <table class="w-w-full">
```

Probably simpler to have the full width as a tailwind class, with the root class moved to the dialog.